index.html

```diff
@@ -69,5 +69,130 @@ <h2 class="connect">LET'S CONNECT</h2>
 				</section>
 				<span class="header-shape"></span>
 			</main>
+      <section class="work">
+        <div class="cardWork sec">
+          <img id="snapshot">
+          <div class="left block_bg">
+            <div class="primary text">
+              <h3 class="project title">Tonic</h3>
+              <div >
+                <ul class="frame">
+                  <li class="client_tag_list"><h class="client_canopy">CANOPY</h></li>
+                  <span class="counter"></span>
+                  <li class="client_tag_role"><h class="client_role_text">Back End Dev</h></li>
+                  <span class="counter_tw"></span>
+                  <li class="client_tag_year"><h class="client_yr_text">2015</h></li>
+                </ul>
+              </div>
+            </div>
+            <div class="client_txt_bd">
+              <p4>A daily selection of privately personalized reads; no accounts or sign-ups required.</p4>
```

Comment:
Kindly, please use an appropriate semantic tag, (P4 does not exist, I suggest putting only P)